odoo/http.py

```diff
@@ -1429,8 +1429,16 @@ def get_response(self, httprequest, result, explicit_session):
         #   (the one using the cookie). That is a special feature of the Session Javascript class.
         # - It could allow session fixation attacks.
         if not explicit_session and hasattr(response, 'set_cookie'):
+            max_age = 90
+            if request.httprequest.path != '/web/database/drop' and request.session.db:
+                configured_max_age = request.env['ir.config_parameter'].sudo().get_param('session.max_age')
+                if configured_max_age:
```

Comment:
We can skip this `if configured_max_age` because if there is not configured_max_age it will be handled in try:..except...